SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -430,7 +430,7 @@ TArray<FWorkerComponentData> EntityFactory::CreateTombstoneEntityComponents(AAct
 TArray<FWorkerComponentData> EntityFactory::CreatePartitionEntityComponents(const Worker_EntityId EntityId,
 																			const InterestFactory* InterestFactory,
 																			const UAbstractLBStrategy* LbStrategy,
-																			VirtualWorkerId VirtualWorker)
+																			VirtualWorkerId VirtualWorker, bool bDebugContexValid)
```

Comment:
clang-format keeps this one line?

Reply:
it's a good line
also nit: 
```suggestion
																			VirtualWorkerId VirtualWorker, bool bDebugContextValid)
```